packages/app/src/cli/prompts/config.test.ts

```diff
@@ -194,12 +202,19 @@ describe('confirmPushChanges', () => {
       const configurationPath = joinPath(tmpDir, 'shopify.app.toml')
       const app = testOrganizationApp({requestedAccessScopes: ['read_products']}) as App
 
+      vi.mocked(fetchAppExtensionRegistrations).mockImplementation(async () => ({
+        app: {
+          extensionRegistrations: [],
+          dashboardManagedExtensionRegistrations: [],
+          functions: [],
+        },
+      }))
```

Comment:
If the mock is the same for all tests, I think is worth moving this to the top to avoid repeating code